test/parallel/xyk-pallet.API.payment-info.test.ts

```diff
@@ -59,24 +77,28 @@ test("GIVEN a paymentInfo request, WHEN extrinsic is sellAsset  THEN zero is ret
   expect(sellAssetPaymentInfo.partialFee).bnEqual(BN_ZERO);
 });
 
-test("GIVEN a paymentInfo request, WHEN extrinsic is multiswapBuyAsset THEN  zero is returned", async () => {
-  const multiswapBuyEvent = api.tx.xyk.multiswapBuyAsset(
-    [GASP_ASSET_ID, KSM_ASSET_ID],
+test("GIVEN a paymentInfo request, WHEN extrinsic is multiswapBuyAsset THEN non-zero is returned", async () => {
+  liqId = await getLiquidityAssetId(GASP_ASSET_ID, ETH_ASSET_ID);
+  const multiswapBuyEvent = api.tx.market.multiswapAsset(
+    [liqId],
+    GASP_ASSET_ID,
     BN_HUNDRED,
+    ETH_ASSET_ID,
     BN_MILLION,
   );
 
   const multiswapBuyPaymentInfo = await multiswapBuyEvent.paymentInfo(
     testUser.keyRingPair,
   );
 
-  expect(multiswapBuyPaymentInfo.partialFee).bnEqual(BN_ZERO);
+  expect(multiswapBuyPaymentInfo.partialFee).bnGt(BN_ZERO);
 });
 
 test("GIVEN a paymentInfo request, WHEN extrinsic is mintLiquidityEvent THEN non-zero is returned", async () => {
-  const mintLiquidityEvent = api.tx.xyk.mintLiquidity(
+  liqId = await getLiquidityAssetId(GASP_ASSET_ID, ETH_ASSET_ID);
+  const mintLiquidityEvent = api.tx.market.mintLiquidity(
```

Comment:
move this to a utility o Market class, so we can reuse it

Reply:
Done